sdk/authorization/ci.yml

```diff
@@ -37,7 +37,7 @@ pr:
       - sdk/authorization/mgmt-
 
 stages:
-  - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  - template: ../../eng/pipelines/templates/stages/archetype-sdk-data.yml
     parameters:
       ServiceDirectory: authorization
       SDKType: data
```

Comment:
Should we set the SDKType in the template now?

Reply:
Good call/catch. I'll remove those from the various ci.yml files.